docs/quick.rst

```diff
@@ -479,7 +480,7 @@ The default error estimation routine is
 
     >>> from sherpa.estmethods import Confidence
     >>> gefit.estmethod = Confidence()
-    >>> print(gefit.estmethod)
+    >>> print(gefit.estmethod)  # doctest: +SKIP
```

Comment:
Why is this being skipped?

Reply:
The output depends on the machine you are running on because the `numcores` lines actually reports the number of cores. I don't see an easy way to inject a fixed number of cores here for this test, neither do I want to replace it with `...`. 

I think that's one of those cases where it's not worth the effort to use that as a test - let's just skip it.

Reply:
I've wondered, too but I'm not sure where to put it. I don't want to pollute the docstring. I'll add a `#  comment` below the docstring.